tests/Unit/SpacesApiTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Storyblok-Api.
+ *
+ * (c) SensioLabs Deutschland <info@sensiolabs.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SensioLabs\Storyblok\Api\Tests\Unit;
```

Comment:
I think we should put those tests into `SensioLabs\Storyblok\Api\Tests\Integration` namespace